__tests__/post.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require("supertest");
+const app = require("../app.js");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection.js");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  describe("Happy path", () => {
+    test("Should add a comment to a review by id", () => {
```

Comment:
Add another test here that checks unnecessary content on the post body is ignored - i.e. {username: "validdoodad", body: "validdoodad", unnecessarykey: "doodad"} - returns 201 with valid comment.